### DISCUSSION TITLE
SQLiteStudio v.3.3.3 : "This app can't run on your PC" (Win10 20H2)

### DISCUSSION THREAD

**Discussion by RuudSchmeitz** [pawelsalawa/sqlitestudio#4101] [Q&A]
Hi Paweł ,

I'm now running SQLiteStudio v3.2.1 (great program BTW !) , and want to update to v.3.3.3 .

This is what I've done :
- downloaded file sqlitestudio-3.3.3.zip
- unpacked the zip-file
- clicked on file SQLiteStudio.exe (in C:\Program Files\SQLiteStudio)

and then I got this error-message :
"This app can't run on your PC
 To find a version for your PC, check with the software publisher."

I'm using a PC with Windows 10 Home, v.20H2 .

BTW :
when I'm running SQLiteStudio v3.2.1 , click on the 'Help' tab , and then on 'Check for updates' , it says :
"You're running the most recent version. No updates are available."

**Comment by pawelsalawa** [comment]
Regarding "no updates" - yes, during work on 3.3.0 version I had to break the updates compatibility, because I dropped "Qt Installer Framework".

Now, regarding you main problem - you wrote you run SQLiteStudio.exe from `C:\Program Files\SQLiteStudio`. Is it possible that you had version 3.2.1 installed in the same directory previously and you overwritten it with version 3.3.3? In that case you might have overwritten some files, while others remain from the old version, thus making your installation part-new, part-old version and causing the problem. I would recommend to first uninstall the 3.2.1 version (either from "Programs and functions" of the Windows, or if it's not there, then just simply delete `C:\Program Files\SQLiteStudio` folder) and only then putting your version 3.3.3 there.

**Reply by RuudSchmeitz** [reply]
"Is it possible that you had version 3.2.1 installed in the same directory previously and you overwritten it with version 3.3.3?"

I don't think so . Before unpacking the **sqlitestudio-3.3.3.zip file** , I renamed `C:\Program Files\SQLiteStudio` into `C:\Program Files\SQLiteStudio - 3.2.1` , and then unpacked the zip-file in a new `C:\Program Files\SQLiteStudio` folder .


"I would recommend to first uninstall the 3.2.1 version (...)"

I checked both options :
**Control Panel / All Control Panel Items / Programs and Features
Settings / Apps / Apps & features**
but there was nothing to uninstall the 3.2.1 version .


But I noticed that the 2 versions (3.2.1 and 3.3.3) have a different number of files . Is that correct ?

![SQLiteStudio - v3 2 1](https://user-images.githubusercontent.com/69636360/117594628-cff59900-b13e-11eb-9202-23335b7b5295.png)

![SQLiteStudio - v3 3 3](https://user-images.githubusercontent.com/69636360/117594644-d84dd400-b13e-11eb-83d6-aa09efb92c08.png)

**Reply by pawelsalawa** [reply]
Different file set is okay.

Do you perhaps have a Qt framework installed somewhere in your system?

**Reply by RuudSchmeitz** [reply]
I'm not sure . How can I find out ?

I checked this page :
https://en.wikipedia.org/wiki/Qt_(software)#Qt_in_use
but I don't use any of these UIs and/or programs .

I **am** a longtime XnView user , but I still use the classical version , not the MP version .

**Reply by pawelsalawa** [reply]
On the second thought - even if you have Qt in the system, it should not impact SQLiteStudio, as it should prioritize the version included in the same directory.

Just to be sure - are you running 64 bit Windows?

**Reply by RuudSchmeitz** [reply]
No , this PC runs the 32-bits version .

![system specs](https://user-images.githubusercontent.com/69636360/117643023-410d6e80-b188-11eb-8f1d-d32094b5b674.png)

**Reply by pawelsalawa** [reply]
This is the reason. Version 3.3.0 moved Windows binaries to 64-bit. For now you're stuck with 3.2.x version.

I've added #4103 to restore 32 bit binaries for both Windows and Linux, but it won't happen fast.

**Reply by RuudSchmeitz** [reply]
"I've added #4103 to restore 32 bit binaries for both Windows and Linux, but it won't happen fast."

OK , thanx for all your replies .
I'll try to be patient .
😎
